src/main.tsx

```diff
@@ -2,14 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './main.css';
 import App from './App';
+import Loading from './components/Loading';
 import { exampleData as data } from './static/data';
 import { Data } from './types';
+import './i18n';
 
-const renderQuizRecap = (id: string, data: Data) => {
+const renderQuizRecap = (id: string, data: Data, locale = 'en') => {
```

Comment:
Can you please document how to configure the language of the component in the `README`?

Reply:
There was already something in the README. I've extended it.